WSSiOS/WSSiOS/Source/Data/Entity/Feed/FeedEntity.swift

```diff
@@ -80,13 +102,17 @@ extension FeedResponse {
 }
 
 struct FeedDetailNovelEntity {
-    let novelId: Int?
-    let novelTitle: String?
-    let novelRating: Float?
-    let hasUserRating: Bool?
-    let novelUserRating: Float?
-    let novelAuthor: String?
-    let novelGenreImage: UIImage?
-    let novelDescription: String?
+    let novelId: Int
+    let novelTitle: String
+    let novelRating: Float
+    let hasFeedAuthorRating: Bool
+    let feedAuthorData: FeedDetailNovelFeedAuthorEntity?
+    let novelGenreImage: UIImage
+    let novelDescription: String
     let novelThumbnailURL: URL?
 }
+
+struct FeedDetailNovelFeedAuthorEntity {
+    let feedAuthor: String
+    let feedAuthorRating: Float
+}
```

Comment:
변수명이 명확하지 않아 보입니다.
해당 구조체는 피드상세뷰 내 novel 컴포넌트에 보여야하는 유저의 닉네임과 평점 같은데 맞을까요?
FeedAuthor라는 네이밍은 유저 (즉,본인) 라고 와닿기 보단 피드를 작성한 외부인처럼 느껴집니다.

```suggestion
struct FeedDetailNovelUserEntity {
    let userNickname: String
    let userRating: Float
}
```

Reply:
느끼신 의미가 맞습니다. 피드를 작성한 유저의 닉네임과 레이팅입니다.

Reply:
네네 네이밍의 기준은 각자마다 다른것이니까요!

Reply:
@Guryss 

> FeedAuthor라는 네이밍은 유저 (즉,본인) 라고 와닿기 보단 피드를 작성한 외부인처럼 느껴집니다.
에서 피드를 작성한 외부인이 맞다는 의미입니다.
계정소유주의 별점이 아닌 피드작성자의 별점을 의미합니다.
따라서 내가 쓴 글이 아닌 타인의 피드를 보았을 때 타인의 평점이 나타납니다 :)